source/initial_temperature/adiabatic_boundary.cc

```diff
@@ -187,13 +117,18 @@ namespace aspect
                                               "adiabatic boundary",
                                               "An initial temperature condition that allows for discretizing "
                                               "the model domain into two layers separated by a user-defined "
-                                              "isothermal boundary using a table look-up approach. The user includes an "
-                                              "input ascii data file that is formatted as 3 columns of `latitude', "
-                                              "`longitude', and `depth', where `depth' is in kilometers and "
-                                              "represents the depth of an initial temperature of 1673.15 K (by default). "
+                                              "isothermal boundary. The user includes an input ascii data file "
+                                              "that is formatted as 3 columns of `longitude(radians)', "
+                                              "`colatitude(radians)', and `depth(meters)', where `depth' represents the depth "
+                                              "of an initial temperature of 1673.15 K (by default). "
+                                              "The first lines in the data file may contain any number of comments if they begin "
+                                              "with `#', but one of these lines needs to contain the number of grid points "
+                                              "in each dimension as for example `# POINTS: 69 121'. Note that the data need "
+                                              "to be sorted in a specific order: the `longitude' coordinate needs to ascend first, "
+                                              "followed by the `colatitude' coordinate, and then the `depth' coordinate at last "
```

Comment:
There is no 'depth' coordinate in your file, at least not in the meaning of this comment, because your grid is only a longitude-colatitude grid (your 'depth' column is a data column that does not need to be ordered). Please adjust comment.